model/task/task.go

```diff
@@ -1501,7 +1501,7 @@ func (t *Task) GetDisplayStatus() string {
 		if !t.Activated {
 			return evergreen.TaskUnscheduled
 		}
-		if t.Blocked() {
+		if t.Blocked() && !t.OverrideDependencies {
```

Comment:
what does OverrideDependencies represent?

Reply:
OverrideDependencies is a flag on the task that gets toggled here. It indicates that the tasks dependencies should be ignored. 
https://github.com/evergreen-ci/evergreen/blob/9dce0b8be4d937b2b6582e0254dc6cf2aecdfb7c/model/task/task.go#L511-L523